docs/chapters/baseplatform/creating-sdk.rst

```diff
@@ -0,0 +1,14 @@
+Building the SDK installer
+===========================
+
+Building the SDK installer is not very different from creating the corresponding
+target image. All you have to do, is pass an additional option to bitbake: 
```

Comment:
I feel that the comma here is in a weird place. "All you have to do is to pass an ..." seems nicer.